impl/sql/mutationstorage/mutation_logs.go

```diff
@@ -28,6 +28,10 @@ import (
 	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
 )
 
+// quantum is the fidelity of the Timestamp column.
+// See https://dev.mysql.com/doc/refman/8.0/en/fractional-seconds.html
```

Comment:
Async: Should `impl/sql` in the directory really be `impl/mysql`? SQL != MySQL, and this may be a trip hazard in the future.

Reply:
yes, that is a change that's needed